test/Models/resnet50v1.mlir

```diff
@@ -119,15 +119,14 @@ func.func @resnet50v1(%arg0: tensor<1x224x224x3xf32>) -> tensor<1x1000xf32> {
   %cst_15 = arith.constant dense<0.000000e+00> : tensor<1x56x56x64xf32>
   %cst_16 = arith.constant dense<0.000000e+00> : tensor<1x112x112x64xf32>
 
-  //
+  // TODO: Why are we checking these?????????
```

Comment:
To add variable checks to the xsmm calls? I think for these whole models we don't really need that detailed checks. We should save the complexity for the `Convert` and `Integration` checks.

Reply:
Most of them are not used (i.e., c0_i64).